ts/src/base/ws/Future.ts

```diff
@@ -1,11 +1,10 @@
 // @ts-nocheck
-
 interface Future extends Promise<unknown> {
     resolve(value: unknown): void;
     reject(reason?: any): void;
 }
 
-export default function Future (): Future {
+export default function CreateFuture (): Future {
```

Comment:
@blair what if we do this?

Reply:
@carlosmiei

Simply renaming `Future` to `CreateFuture` doesn't fix the issue.

The `d282f67` does fix it.

For style, why not keep the `Future` type as `Future` since it's like `Promise` and rename `Future ()` to `createFuture ()` with a lowercase `c`.

I'm pushing an updated version shortly for you to review.